addons/product/models/product_template.py

```diff
@@ -172,6 +172,7 @@ def _read_group_categ_id(self, categories, domain):
     product_tag_ids = fields.Many2many(
         string="Tags", comodel_name='product.tag', relation='product_tag_product_template_rel'
     )
+    is_dynamically_created = fields.Boolean("Is Dynamically Created", default=False, compute='_compute_is_dynamically_created')
```

Comment:
why the default ? I doubt it's necessary
The field could be moved after `has_configurable_attributes`.